source/NVDAObjects/__init__.py

```diff
@@ -1433,3 +1434,11 @@ def getSelectedItemsCount(self,maxCount=2):
 		For performance, this method will only count up to the given maxCount number, and if there is one more above that, then sys.maxint is returned stating that many items are selected.
 		"""
 		return 0
+
+	#: Type definition for auto prop '_get_isAboveLockScreen'
+	isAboveLockScreen: bool
+
+	def _get_isAboveLockScreen(self) -> bool:

```

Comment:
I'd prefer to see this property reversed / renamed to isBelowLockScreen as I think it is easier to understand if it is about being locked because the lock screen is there and we are below it, as opposed to the lock screen is not there or the lock screen is there and we are above it.
As in, being locked (and therefore probably limited) is the exception, the majority case is to not be locked (either because the machine is not locked, or we are above the lock screen).

Reply:
Makes sense, I've changed this where relevant across this PR.